locationConfig.json

```diff
@@ -16,7 +16,7 @@
         "details": {
             "connector": {
                 "sproxyd": {
-                    "bootstrap": ["localhost:8182"]
+                    "bootstrap": ["localhost:8182:/proxy/arc/"]
```

Comment:
I believe its parsed in this method: https://github.com/scality/sproxydclient/blob/master/lib/sproxyd.js#L45